### ISSUE TITLE
entc/gen: runtime/runtime.go generates wrong UUID import depending on module cache state

### ISSUE THREAD

**Issue by mgabeler-lee-6rs** [facebook/ent#891]
This is basically a rehash of #773 but with more detail and a minimal repro repo attached.

Setup:
* Make a minimal module using `ent`
* Make a schema that has a UUID field using `github.com/google/uuid`
* Add a hook to the schema

Works:
* Clean your module cache: `go clean -modcache`
* Run ent code generation
* Build your project

Fails:
* Seed your module cache with `github.com/gofrs/uuid`:
  * E.g., in a separate shell, `cd $(mktemp -d) && go mod init test && go get github.com/gofrs/uuid`
  * Note that this is not changing the module under test _at all_!
* Re-run code generation & build

Demo repo ready for the above: https://github.com/mgabeler-lee-6rs/ent-uuid-demo

Build error:
```
ent/runtime/runtime.go:22:20: cannot use exampleDescID.Default.(func() "github.com/gofrs/uuid".UUID) (type func() "github.com/gofrs/uuid".UUID) as type func() "github.com/google/uuid".UUID in assignment
```

----

In practice this is biting me extra because my project uses https://github.com/jackc/pgx which internally uses the problem UUID library, so even cleaning my module cache constantly won't actually work around the issue for me. For the moment I'm having to resort to a `go:generate` step that patches up `runtime.go` :frowning_face: 

Using `pgx` isn't necessary to repro this, however. Just having _some_ project on your machine that uses `gofrs/uuid` so that it ends up in the module cache is enough to cause this failure.

It sort of looks like the template for the generation of `ent/runtime/runtime.go` is trying to copy the imports from the schema file, but I may be misreading it, or it may simply not be working. VSCode's editor undo state sometimes captures some intermediate state of the generated `runtime.go` file after initial generation but before `goimports` is run, and it doesn't show any `uuid` import then.

**Comment by a8m** [comment]
Hey @mgabeler-lee-6rs and thanks for reporting this issue.

What version do you use for ent? It should be fixed in the latest version.

Can you please update to latest master and report if you still get this

```sh
go get github.com/facebook/ent@master 
```

**Comment by mgabeler-lee-6rs** [comment]
I'm using the latest release -- v0.4.3 -- but the problem reproduces exactly the same on `master`. I've updated my repro repo to use latest master so you can see for yourself. Please take a look (the repo is _miniscule_) and try the steps out for yourself to see.

**Comment by a8m** [comment]
Thanks for reporting this issue with a proper reproducible example. It helped me tackling this quickly.

Fixed via #901.

**Comment by mgabeler-lee-6rs** [comment]
Thank you much for the quick fix :heart: 

Confirmed it fixes my "real" app in addition to the repro demo :smiley:
